src/components/Post/Post.tsx

```diff
@@ -99,6 +100,7 @@ const Post: FC<IPostProps> = (props) => {
 
 	const isOnchainPost = checkIsOnChainPost(proposalType);
 	const isOffchainPost = !isOnchainPost;
+	const [lastVote, setLastVote] = useState< ILastVote | undefined >(undefined);
```

Comment:
Why is the `lastVote` state stored here ? It's not being used here neither we're setting via `setLastVote` in this page.
It's only being passed to GovernanceSidebar so it should be in GovernanceSidebar

Reply:
when lastVote was in Governance side bar , it was causing an issue , state was not accessible when we were switching to mobile view from web view . to solve this issue moved lastVote state to the parent of Gov sidebar